### PR TITLE
added babylon again w/o subGit

### PR DESCRIPTION
**Pull request by t0dorakis**
Awesome you want to make a PR  
please see [https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md](https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md) for further infos.

### CODE REVIEW

**Comment by fabianmoronzirfas** [comment]
There is a problem with your folder structure

1. The files `splash.png` and `thumb.png` (not ~~splash-thumb.png~~) should be on the same level as the index.md file. 
2. Also don't introduce a new folder structure for your images. Just add your images into the `assets/images` folder and not into the `assets/visuals`.  
3. Make the folder structure flat. no subfolders with `assets/visuals/it1/1.jpg`. Just add the iteration to the filename `assets/images/iteration1-1.jpg`

**Comment by fabianmoronzirfas** [comment]
Yo deleted the file `projects/index.md` that is bad. Please add it again https://github.com/fabianmoronzirfas/gestalten-in-code/pull/59/files#diff-6087c0ef875554f4409ac52928d79279

File: projects/syntheticBabylon/index.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+authors:
+  - Theodor Hillmann
+authors-url:
+  - "http://theodorhillmann.de"
+layout: project
+title: "Synthetic Babylon: analogue generative architecture"
+type: project
+---
+
+# Synthetic Babylon
```
Comment (fabianmoronzirfas) [inline projects/syntheticBabylon/index.md]: The title is added by the system. No need to add it again

File: projects/syntheticBabylon/index.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+## analogue generative architecture
+
+This work is part of a class at the University of Applied Sciences Potsdam. The project engages in analogue generative design. Initially an analogue algorithm has been written. I tried to generate architectural sketches inspired by Constant's "New Babylon". This initially analog algorithm has been extended and developed over many iterations and finally put in form.
+
+For the algorithms first iteration pencils and sheets of paper were handed to 5 people. They were instructed to consequently follow the rules of the algorithm listed below. The following "code" is kept in written-german to stay closest to the original experience.
+
+### Iteration 1:
+
+Arbeite die folgenden Punkte nacheinander ab.
```
Comment (fabianmoronzirfas) [inline projects/syntheticBabylon/index.md]: Can you please translate the algorithms into English

File: projects/syntheticBabylon/index.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+Nutze dafür ein blankes Blatt Papier und ein Bleistift (wenn
+möglich).
+
+1. Zeichne zwei Punkte die waagerecht zueinander stehen, und
```
Comment (fabianmoronzirfas) [inline projects/syntheticBabylon/index.md]: Maybe also make them a Markdown quote. All the other students did this as well. Just add to each line at the beginning a 

    >

File: projects/syntheticBabylon/index.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+5. Verbinde die entstandenen oberen Berührungspunkte auf den
+Schrägen mit den noch nicht verbundenen Punkt.
+6. Setze diese Technik fort bis ein Gefühl von Stadtraum entsteht
+
+Vielen Dank!
+
+The results turned out to be rough and did not fulfil the visual expectations.
+Therefore in the following steps the algorithm has continuously been adjusted to the impressions of natural and organic structures.
+
+![](https://github.com/t0dorakis/syntheticBabylon/blob/master/assets/visuals/IT1/1.png)
```
Comment (fabianmoronzirfas) [inline projects/syntheticBabylon/index.md]: No absolut links to your repo please. When you updated the folder structure you can link them like this:

File: projects/syntheticBabylon/index.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+eine schwebende Stadt erschaffen?
+
+Vielen Dank!
+
+![](https://github.com/t0dorakis/syntheticBabylon/blob/master/assets/visuals/IT3/1.png)
+![](https://github.com/t0dorakis/syntheticBabylon/blob/master/assets/visuals/IT3/2.png)
+![](https://github.com/t0dorakis/syntheticBabylon/blob/master/assets/visuals/IT3/3.png)
+
+
+The picture underneath is my favorite one and it is closest to what I tried to achieve.
```
Comment (fabianmoronzirfas) [inline projects/syntheticBabylon/index.md]: Can you write a bit more conclusion? Not only I like the last one. Did you get anything out of this exercise or was it plain stupid to you?

**Comment by fabianmoronzirfas** [comment]
🤣  you had a notification while doing your screenshots

**Comment by fabianmoronzirfas** [comment]
Merged

**Comment by t0dorakis** [comment]
supi!

**Comment by fabianmoronzirfas** [comment]
And live https://interface.fh-potsdam.de/gestalten-in-code/projects/syntheticBabylon/

**Pull request merged**